travis/setup_vtk_env.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then 
+    if [[ "${VTK_VERSION}" == "5.8" ]]; then 
+        # With vtk 5, we install the 5.8 standard we get from brew
+        brew install vtk5
+        export PYTHONPATH=/usr/local/opt/vtk5/lib/python2.7/site-packages
+        export DYLD_LIBRARY_PATH=/usr/local/opt/vtk5/lib
+    elif [[ "${VTK_VERSION}" == "6.3" ]]; then 
+        # unfortunately we can't do this, for the following reasons:
+        #  - brew does not provide VTK 6, so we would have to have to download it
+        #  - unfortunately, it's built against the Apple python, so we would have 
+        #    to change the dependency path for the python library with install_name_tool
+        #  - also unfortunately, it's built against python 2.6, so we can't use it against
+        #    libpythn2.7
+        echo "Cannot work against VTK 6.3. VTK provided links against incorrect python version"
+        exit 1
+    elif [[ "${VTK_VERSION}" == "7.0" ]]; then 
+        # Brew has vtk 7 at this handle
+        brew install vtk
+        # but apparently binds against the system framework, hardcoded,
+        # so we need to change it manually.
+        for f in `find /usr/local/Cellar/vtk -name *.dylib`; do
+            install_name_tool -change /System/Library/Frameworks/Python.framework/Versions/2.7/Python /usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/Python $f
+        done
+        for f in `find /usr/local/Cellar/vtk -name *.so`; do
+            install_name_tool -change /System/Library/Frameworks/Python.framework/Versions/2.7/Python /usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/Python $f
+        done
+        export PYTHONPATH=/usr/local/opt/vtk/lib/python2.7/site-packages
+        export DYLD_LIBRARY_PATH=/usr/local/opt/vtk/lib
+    fi
+else
+    # Linux
+    if [[ "${VTK_VERSION}" == "5.8" ]]; then
+        # Do nothing. 5.8 is already on the system
+        :
+    else
+        # This is the name of the directory after unpacking
+        VTK_PYTHON=VTK-${VTK_VERSION}.0-Linux-64bit
```

Comment:
Perhaps we could modify .travis.yml to pass VTK_VERSION = 7.0.0 or 6.3.0 so that we can skip the `.0` after `${VTK_VERSION}`
And perhaps the line in `.travis.yml` that print the VTK version should really exit with 1, prevent us from thinking the tests were run with the right version when it is not.
i.e.  Instead of
```
# .travis.yml
python -c "from __future__ import print_function; import vtk; print('VTK Version:', vtk.vtkVersion.GetVTKVersion())"
```
Do
```
# python script
from __future__ import print_function
import vtk
import sys
vtk_version = vtk.vtkVersion.GetVTKVersion()
desired_version = os.environ.get('VTK_VERSION')
if vtk_version != desired_version:
    print('Loaded VTK version is', vtk_version, 'but you wanted', desired_version)
    sys.exit(1)
```
my bad, i wrote that thing.